scripts/aggregate-translations/aggregate-messages.js

```diff
@@ -7,17 +7,15 @@ const aggregateTranslationMessages = (translationDirectories, locales) => {
     translations[language] = {};
   });
 
-  translationDirectories.forEach(dir =>
-    // Check the directory for a translation file for each locale
```

Comment:
It might be helpful to keep this comment but I don't have a strong opinion

Reply:
I added the comment back in.